omero_certificates/certificates.py

```diff
@@ -132,8 +132,15 @@ def create_certificates(omerodir):
     #
     # See:
     #   * https://www.rfc-editor.org/rfc/rfc7919.txt
-    with open(grouppath, "w") as pem:
-        pem.write(FFDHE2048_PEM)
+    pem_exists = False
+    if os.path.exists(grouppath):
+        log.info("Using existing ffdhe2048.pem")
+        with open(grouppath, "r") as pem:
+            if pem.read() == FFDHE2048_PEM:
+                pem_exists = True
+    if not pem_exists:
+        with open(grouppath, "w") as pem:
+            pem.write(FFDHE2048_PEM)
```

Comment:
As a mirror to the log statement discussed above about re-using the existing PEM file, I would propose to add a logging statement above similar to the ones below e.g.

```
log.info("Creating PEM file with pre-defined DH group: %s", pem)
```